crud.py

```diff
@@ -139,24 +172,26 @@ async def delete_market_zone(zone_id: str) -> None:
 async def create_market_stall(data: createStalls) -> Stalls:
     stall_id = urlsafe_short_hash()
     await db.execute(
-        """
+        f"""
         INSERT INTO market.stalls (
             id,
             wallet,
             name,
             currency,
             publickey,
+            privatekey,
```

Comment:
- why does a `stall` need a `privatekey`?
- I thought there is one key per merchant

Reply:
we either have it there or we need a global "settings" for merchant with the keypair...